src/Http/OfferRestBaseController.php

```diff
@@ -82,30 +82,6 @@ public function deleteTypicalAgeRange(string $cdbid)
         return new NoContent();
     }
 
-    public function updateOrganizer(string $cdbid, string $organizerId): Response
-    {
-        $this->editor->updateOrganizer($cdbid, $organizerId);
-
-        return new NoContent();
-    }
-
-    /**
-     * @deprecated
-     */
-    public function updateOrganizerFromJsonBody(Request $request, string $cdbid): Response
```

Comment:
This method is still used for an even older route

Reply:
See https://github.com/cultuurnet/udb3-silex/blob/master/app/Event/EventControllerProvider.php#L77
That route is actually better in terms of URL because the organizer id should not be part of the URL, but it uses the wrong HTTP method sadly. 😄